modules/constructor/constructor-front-pocket/constructor-front-pocket.model.js

```diff
@@ -1,17 +1,21 @@
 const mongoose = require('mongoose');
 const Language = require('../../../models/Language').schema;
 const CurrencySet = require('../../../models/CurrencySet').schema;
-const ImageSet = require('../../common/ImageSet').schema;
 
 const constructorFrontPocketSchema = new mongoose.Schema({
   name: [Language],
   material: {
     type: mongoose.Schema.Types.ObjectId,
     ref: 'Material',
   },
+  color: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Color',
+  },
   image: String,
   basePrice: [CurrencySet],
-  available: Boolean
+  available: Boolean,
+  default:Boolean,
```

Comment:
same

Reply:
Thit is default as a key in order you want use this element by defauld when counstructor is loaded for the first time